lib/async/io/ssl_endpoint.rb

```diff
@@ -29,10 +29,10 @@ def initialize(endpoint, **options)
 				
 				@endpoint = endpoint
 				
-				if ssl_context = options[:ssl_context]
-					@context = build_context(ssl_context)
+				@context = if ssl_context = options[:ssl_context]
```

Comment:
I personally think this change makes readability worse. I don't like using `if` expressions because control flow is difficult to follow.

Reply:
Changed back.

Thanks for bringing out your perspective on style, with arguments for _why_ you prefer one way over another. That takes care - I'm grateful.